### DISCUSSION TITLE
`ros-recv` not available in Termux

### DISCUSSION THREAD

**Discussion by MatthewScholefield** [roc-streaming/roc-toolkit#416] [Q&A] [answered]
From the [tutorial](https://gavv.github.io/articles/roc-tutorial/#android-termux) it sounds like after installing `roc` with Termux we should be able to run `roc-recv`. However, neither `ros-recv` nor `ros-send` appear to be executables. Is there something wrong or is `roc-recv` not necessary on Android?

Thanks,
Matthew

**Comment by gavv** [comment, marked as answer]
Hi, thanks for interest in the project.

We don't have command-line tools for Android yet because of these two issues: #232, #246. They're to be resolved in future.

Currently we offer two options for Android:

* PulseAudio sink-input (acting as Roc receiver), can be used with Termux
* native [Android app](https://github.com/roc-streaming/roc-droid), currently in prototype stage; currently it also supports only receiving

**Comment by devilsvenom** [comment]
I trying to install roc library
"$ pkg install roc"
But it shows error
"Unable to locate pkg roc"

**Comment by gavv** [comment]
They moved it to disabled packages: https://github.com/termux/termux-packages/tree/master/disabled-packages/roc
I don't know the reason
